src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -3675,6 +3884,41 @@ components:
             deciding_factors:
               $ref: "#/components/schemas/AccountDepositPreValidationDecidingFactors"
 
+    TransactionSubintentStatusRequest:
+      allOf:
```

Comment:
MINOR: Can we get rid of the all-of if it's just one item? Or is there a reason for doing it this way?

Reply:
yes we can, i think i left it by accident once i've removed `at_ledger_state` param.